ide/api.java.classpath/src/org/netbeans/api/java/classpath/ClassLoaderSupport.java

```diff
@@ -90,7 +90,7 @@ private ClassLoaderSupport (final ClassPath cp, final ClassLoader parentClassLoa
      * @throws ClassNotFoundException
      */
     @Override
-    protected Class findClass (String name) throws ClassNotFoundException {
```

Comment:
Please don't change signature from raw type to generic type without a clear plan. This can introduce compatibility issues (I suffered from it in another project).

Reply:
@matthiasblaesing ah, yes, thanks for that reminder! :smile:  From me, -1 to this PR as is.  Maybe split into two?  A PR changing internal method variables should be fairly safe?  Then another for considering method arguments and return values where appropriate / with a plan?

Reply:
So it sounds like I should just split this into 2 changes.. 1 - method variables and 2 - method signature.. Agreed that #2 needs some planning and thought but #1 should be pretty strait forward..

Correct?

Reply:
Yes, splitting this and maybe just doing the non-API parts right now sound like a good plan.